ionic2/src/app/app.component.ts

```diff
@@ -1,21 +1,29 @@
-import { Component } from '@angular/core';
-import { Platform } from 'ionic-angular';
+import { Component, ViewChild } from '@angular/core';
+import { Platform, Nav } from 'ionic-angular';
 import { StatusBar } from 'ionic-native';
 import 'rxjs/add/operator/map';
 
 import { MapPage } from '../pages/map/map.page';
+import { NotificationService } from '../services/notification.service';
 
 @Component({
   templateUrl: './app.html'
 })
 export class App {
   rootPage = MapPage;
 
-  constructor(private platform: Platform) {
+  @ViewChild('content') nav: Nav;
+
+  constructor(private platform: Platform, private notificationService: NotificationService) {
     platform.ready().then(() => {
-      // Okay, so the platform is ready and our plugins are available.
-      // Here you can do any higher level native things you might need.
-      StatusBar.styleDefault();
+      if (platform.is('cordova')) {
+        StatusBar.styleDefault();
+      }
+
+      if (platform.is('android')) {
```

Comment:
`cordova` is also true if we are running the app inside the browser so it's probably not what we want.
The decision to only publish on Android was announced by Guy or one of the tutors during the presentation week. I tried looking for some official statement but couldn't find anything unfortunately :confused: